### DISCUSSION TITLE
Disable Update Message

### DISCUSSION THREAD

**Discussion by j-manu** [wezterm/wezterm#3545] [General]
Whenever I open a new tab I get a message about the latest update
<img width="285" alt="CleanShot 2023-04-17 at 15 59 29@2x" src="https://user-images.githubusercontent.com/72501/232459063-0a95ed6d-ce6c-4036-a1c8-61d28c55f86a.png">

How do I disable this?

**Comment by nekowinston** [comment]
[`config.show_update_window = false`](https://wezfurlong.org/wezterm/config/lua/config/show_update_window.html)

**Reply by ShroomiePerson** [reply]
i'm not experienced with like, programming in lua (or programming in general) or working with the terminal as much as most who are more linux-savy than i, is this right? if it is, why might it not be removing the update message?
![Screenshot_20240706_134934](https://github.com/wez/wezterm/assets/129551111/a789d09f-0dbd-4541-ae50-6b695ee9d96e)

**Reply by ShroomiePerson** [reply]
like, i'm better at "linux-ing" than your average user (as evidenced by the fact that i like touching my terminal, among other things), but i'm still hardly very good.

**Comment by j-manu** [comment]
Thank you. I was able to find more config here - https://wezfurlong.org/wezterm/config/lua/config/index.html

@wez It will be great if Reference can be added to the left menu on Configuration Page.  It was not immediately apparent to me that Lua Reference has the config reference


<img width="309" alt="CleanShot 2023-04-18 at 12 10 20@2x" src="https://user-images.githubusercontent.com/72501/232692623-da79098e-323c-4f2f-af6a-78be08ef9a63.png">

**Reply by wez** [reply]
There is a link on the other side of the page:
<img width="294" alt="image" src="https://user-images.githubusercontent.com/117777/232817270-0c867a63-f677-404a-986e-217fee5e24fb.png">

I did try to add a link to the left, but mkdocs only allows a single entry for a given page.

I opted to rename "Lua Reference" to "Full Config & Lua Reference" and move the "struct: Config" item to the top of the nav in that section, and rename the nav entry to "Config Options" in 57f07083c7ee398f01ee5cb9b85daa04f058eb64

**Reply by j-manu** [reply]
I missed the TOC on the right. What do you think about integrating the TOC with left navigation - https://squidfunk.github.io/mkdocs-material/setup/setting-up-navigation/#with-navigation-integration ?

**Reply by wez** [reply]
the navigation on the left is already pretty crowded, especially in the reference section, so I prefer to have the current page toc on the right to make it less overwhelming.
